content/blog/chif-driver-not-found.md

```diff
@@ -0,0 +1,115 @@
+tags:
+  - ilo-restful-api
+  - redfish
+  - chif driver
+  - ilorest
+  - ilo 5
+---
+## Introduction
+
+When using the HPE RESTful Interface Tool ([iLOrest](http://hpe.com/info/resttool)) or other applications like the Smart Update Manager ([SUM](https://www.hpe.com/us/en/servers/smart-update.html)) or the integrated Smart Update Tool ([iSUT](https://support.hpe.com/hpesc/public/docDisplay?docLocale=en_US&docId=emr_na-a00068223en_us)) you may get errors similar to: `Chif driver not found, please check that the chif driver is installed`, without any other information to identify the origin of the problem or to fix it. This article attempts to explain when this error occurs and why.
+
+## What is the CHIF driver ?
+
+The Channel Interface (CHIF) driver is an HPE proprietary driver allowing  in-band communication between applications and the Insight Lights Out (iLO) within HPE iLO based servers. In the details, it is a dynamic loadable library (`.dll`) on Windows and a shared object (`.so`) on Linux.
+
+This driver can only be loaded by applications when an iLO chip set is embedded in the computer.
+
+![The CHIF driver allows communication between applications and iLO](/img/chifdriver.png "The CHIF driver allows communication between applications and iLO")
+
+The CHIF driver is packaged with HPE applications requiring it and you don't need to import it manually. However, for home grown applications using the HPE Redfish library or PowerShell Cmdlets (i.e. [HPESysinfoCmdlets](https://www.powershellgallery.com/packages?q=HPESysinfoCmdlets)), you will have to  install it from HPE Software ProLiant Pack (SPP) or download it from the [HPE Support Center](https://internal.support.hpe.com/hpesc/public/km/search#q=ilo%205%20channel%20interface%20driver&t=DriversandSoftware&sort=relevancy&numberOfResults=25) before a manual installation if not already installed by an application requiring it. 
+
+## Quick reminder of iLOrest in-band management
+
+The [iLOrest](http://hpe.com/info/resttool) tool allows in-band and out-of-band Redfish management as explained in [this article](https://developer.hpe.com/blog/managing-ilo-sessions-with-redfish/).
+
+If you are logged in an HPE iLO 4 or iLO 5 based server as a privileged user (`root` or `Administrator`), you can perform in-band management tasks with iLOrest without providing any credentials. You just have to issue GET or SET commands in a straight forward manner to query the local iLO Redfish service.
+
+The following picture shows an SSH root session creation, toward an HPE DL360 Gen10 with the execution of an iLOrest GET command without any formal login process. The `Discovering data...Done` message means that iLOrest could effectively log into the local iLO 5 and save metadata in its cache. Then, it displays the output of the GET command (`Model=iLO 5`).
+
+![Successful in-band GET command](/img/successfulinbandget.png "Successful in-band GET command")
+
+## When do you get the Chif Driver not found error ?
```

Comment:
Because there is still that space between "error" and the exclamation mark, it pushes the exclamation mark to the next line all by itself. You might want to fix this...